### PR TITLE
Update quantization performance data with VNNI

### PR DESCRIPTION
**Pull request by xinyu-intel**
Update quantization performance data collected on AWS c5.24xlarge.

### CODE REVIEW

**Comment by xinyu-intel** [comment]
@pengzhao-intel

**Comment by mli** [comment]
Job PR-838-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-838/1/index.html 
Code coverage of this PR:  vs. Master:

File: docs/tutorials/deployment/int8_inference.py [hunk 1]
```diff
@@ -18,28 +18,28 @@
...
+.. figure:: https://user-images.githubusercontent.com/17897736/60255306-62129200-9884-11e9-96de-3f145be70431.png
    :alt: Gluon Quantization Performance
 
    Gluon Quantization Performance
 
 +-----------------------+----------+------------+------------------+------------------+---------+-----------------+-----------------+
 |  Model                | Dataset  | Batch Size | C5.18xlarge FP32 | C5.18xlarge INT8 | Speedup | FP32 Accuracy   | INT8 Accuracy   |
 +=======================+==========+============+==================+==================+=========+=================+=================+
-| ResNet50 V1           | ImageNet | 128        | 122.02           | 276.72           | 2.27    | 77.21%/93.55%   | 76.86%/93.46%   |
+| ResNet50 V1           | ImageNet | 128        | 191.17           | 1384.4           | 2.27    | 77.21%/93.55%   | 76.08%/93.04%   |
```
Comment (pengzhao-intel) [inline docs/tutorials/deployment/int8_inference.py]: change tilte of this table to 24Xlarge

Comment (xinyu-intel) [reply docs/tutorials/deployment/int8_inference.py]: done

File: docs/tutorials/deployment/int8_inference.py [hunk 2]
```diff
@@ -50,7 +50,7 @@
 .. code:: bash
 
    # with Pascal VOC validation dataset saved on disk
-   python eval_ssd.py --network=vgg16_atrous --quantized --data-shape=300 --batch-size=224 --dataset=voc
+   python eval_ssd.py --network=vgg16_atrous --quantized --data-shape=300 --batch-size=224 --dataset=voc --benchmark
```
Comment (pengzhao-intel) [inline docs/tutorials/deployment/int8_inference.py]: Add OMP environment before comment?

Comment (xinyu-intel) [reply docs/tutorials/deployment/int8_inference.py]: ok

**Comment by mli** [comment]
Job PR-838-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-838/2/index.html 
Code coverage of this PR:  vs. Master:

File: docs/tutorials/deployment/int8_inference.py [hunk 3]
```diff
@@ -18,28 +18,28 @@
...
-| SSD-VGG 512*          | VOC      | 224        | 7.63             | 11.69            | 1.53    | 78.41           | 78.39           |
+| SSD-VGG 512*          | VOC      | 224        | 6.78             | 37.62            | 1.53    | 78.41           | 78.38           |
 +-----------------------+----------+------------+------------------+------------------+---------+-----------------+-----------------+
-| SSD-resnet50_v1 512*  | VOC      | 224        | 17.81            | 34.55            | 1.94    | 80.21           | 80.16           |
+| SSD-resnet50_v1 512*  | VOC      | 224        | 28.59            | 143.7            | 1.94    | 80.21           | 80.25           |
 +-----------------------+----------+------------+------------------+------------------+---------+-----------------+-----------------+
-| SSD-mobilenet1.0 512* | VOC      | 224        | 31.13            | 48.72            | 1.57    | 75.42           | 75.04           |
+| SSD-mobilenet1.0 512* | VOC      | 224        | 65.97            | 212.59           | 1.57    | 75.42           | 74.70           |
```
Comment (pengzhao-intel) [inline docs/tutorials/deployment/int8_inference.py]: Did you update the speedup in the table?

**Comment by mli** [comment]
Job PR-838-6 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-838/6/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-838-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-838/7/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-838-8 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-838/8/index.html 
Code coverage of this PR:  vs. Master:

**Comment by pengzhao-intel** [comment]
@zhreshold @xinyu-intel could you help to merge this PR? thanks:)

**Pull request merged**